cpp/lazperf/lazperf.cpp

```diff
@@ -525,6 +525,9 @@ las_compressor::ptr build_las_compressor(OutputCb cb, int format, size_t ebCount
         break;
     case 8:
         compressor.reset(new point_compressor_8(cb, ebCount));
+        break;
+    default:
+        std::cerr << "Invalid/unsupported format " << format << ".\n";
```

Comment:
Should this throw or do some other kind of error handling?

Reply:
Not sure, which is why it just prints an error. The caller gets a nullptr so should be able to detect and handle. 🤷

Reply:
ok, but libraries shouldn't just print random stuff to stderr/stdout IMO.